src/plugins/apollo/screens/poll/Form.jsx

```diff
@@ -1,23 +1,29 @@
 import PropTypes from 'prop-types';
-import React, { Fragment } from 'react';
+import React from 'react';
 import { reduxForm } from 'redux-form';
 
 import AdvancedFields from '@triniti/cms/plugins/common/components/advanced-fields';
 import History from '@triniti/cms/plugins/pbjx/components/history';
 import Message from '@gdbots/pbj/Message';
 import PollFields from '@triniti/cms/plugins/apollo/components/poll-fields';
 import RawContent from '@triniti/cms/components/raw-content';
-import StreamId from '@gdbots/schemas/gdbots/pbjx/StreamId';
 import TaxonomyFields from '@triniti/cms/plugins/taxonomy/components/taxonomy-fields';
+import setBlocks from '@triniti/cms/plugins/blocksmith/utils/setBlocks';
 
 import schemas from './schemas';
 
-const Form = ({ isEditMode, node, tab }) => {
-  const streamId = StreamId.fromString(`poll.history:${node.get('_id')}`);
-
+const Form = ({ isEditMode, form, node, tab }) => {
   switch (tab) {
     case 'history':
-      return <History schema={schemas.getNodeHistoryRequest} streamId={streamId} />;
+      return (
+        <History
+          isEditMode={isEditMode}
+          formName={form}
+          node={node}
+          schema={schemas.getNodeHistoryRequest}
+          setBlocks={setBlocks}
```

Comment:
@rsumilang polls dont have blocks so i dont think you need to provide this. can just default to noop